RIPS/rip-77XX.md

```diff
@@ -2,7 +2,7 @@
 rip: 77XX
 title: Preinstall deterministic deployment factories
 description: Proposal to add deployment factory contracts at common addresses to enable deterministic contract deployments
-author: Richard Meissner (@rmeissner)
+author: Richard Meissner (@rmeissner), Mikhail Mikheev (@mmv08), Nicholas Rodrigues Lordello (@nlordell)
```

Comment:
I'm not an author yet :P

Reply:
You made a bunch of good suggestions and reviewed some ideas, I reckon that counts